lib/beaker/subcommands/subcommand_util.rb

```diff
@@ -117,6 +129,45 @@ def self.with_captured_output
           $stderr = old_stderr
         end
       end
+
+      def self.store
+        @@store
+      end
+
+      # Load the beaker config
+      def self.load_config()
```

Comment:
This seems incorrectly named; it seems to initialize the file_path and @@store object, but it doesn't actually load anything, per se...